src/neuroconv/datainterfaces/behavior/fictrac/fictracdatainterface.py

```diff
@@ -0,0 +1,523 @@
+            "This is the direction the animal is moving in the lab frame. "
+            "add to animal heading to get direction in the world."
+            "This values is inferred by the rotation of the ball (roll and pitch)"
+        )
+
+        spatial_seriess_kwargs = dict(
+            name="SpatialSeriesMovementDirection",
+            reference_frame=reference_frame,
+            data=movement_direction,
+            unit="radians",
+            description=description,
+        )
+
+        if write_timestamps:
+            spatial_seriess_kwargs["timestamps"] = timestamps
+        else:
+            spatial_seriess_kwargs["rate"] = rate
+
+        spatial_series = SpatialSeries(**spatial_seriess_kwargs)
+        compass_direction_container.add_spatial_series(spatial_series)
+
+        # Add movement speed
+        movement_speed = fictrac_data_df["movement_speed"].values
+        reference_frame = "lab"
+        description = "Instantaneous running speed (radians/frame) of the animal."
+
+        spatial_seriess_kwargs = dict(
+            name="SpatialSeriesMovementSpeed",
+            data=movement_speed,
+            reference_frame=reference_frame,
+            unit="radians",
+            description=description,
+        )
+
+        if write_timestamps:
+            spatial_seriess_kwargs["timestamps"] = timestamps
+        else:
+            spatial_seriess_kwargs["rate"] = rate
+
+        spatial_series = SpatialSeries(**spatial_seriess_kwargs)
+        compass_direction_container.add_spatial_series(spatial_series)
+
+        # Add position in radians from the lab
+        position_lab_columns = [
+            "x_pos_radians_lab",
+            "y_pos_radians_lab",
+        ]
+
+        description = (
+            "x and y positions in the lab frame in radians. These values are inferred by integrating "
+            "the rotation of the across time. "
+        )
+
+        df_lab_position = fictrac_data_df[position_lab_columns]
+        data = df_lab_position.to_numpy()
+        reference_frame = "lab"
+        spatial_seriess_kwargs = dict(
+            name="SpatialSeriesPosition",
+            data=data,
+            reference_frame=reference_frame,
+            unit="radians",
+            description=description,
+        )
+
+        if write_timestamps:
+            spatial_seriess_kwargs["timestamps"] = timestamps
+        else:
+            spatial_seriess_kwargs["rate"] = rate
+
+        spatial_series = SpatialSeries(**spatial_seriess_kwargs)
+        compass_direction_container.add_spatial_series(spatial_series)
+
+        # Add integrated motion
+        integrated_motin_columns = [
+            "forward_motion_lab",
+            "side_motion_lab",
+        ]
+
+        description = (
+            "Integrated x/y position (radians) of the sphere in laboratory coordinates neglecting"
+            " heading. Equivalent to the output from two optical mice"
```

Comment:
Not entirely sure what is meant by the last part of this description, can you elaborate?

Reply:
Imagine the animal looking in a specific direction (that is the heading). Rotations about the z (yaw) axis of the animal can be integrated to obtain the animal’s heading direction in the current time frame. So you have the direction that the animal is moving towards.
Rotations in the two other two remaining axes can be give you the instataneous speed and if you integrate that in the direction of the heading get the x and y positions

![image](https://github.com/catalystneuro/neuroconv/assets/6429509/682ddfe4-1a35-474d-a917-dd5ef41e828b)

(from the paper).

If you only integrate the rotations in the other remaining axis (wx and xy) you get the the position of the ball as it moves disregarding the heading. 

The reference to two optical mice comes from the documentation:

https://github.com/rjdmoore/fictrac/blob/9ac055e52d89f49f492a8eb4e1f7c5b8cd6df40a/doc/data_header.txt#L33-L36

And I think is added because capturing the original paper compares their estimation method to other methods that use two sensors from optical mice and they included that variable in the output for that comparison.

Reply:
Sorry I should have been more specific - the first sentence ('Integrated x/y position (radians) of the sphere in laboratory coordinates neglecting heading.') make sense

What I'm saying is the last statement 'Equivalent to the output from two optical mice' seems tacked on and and not defined what an 'optical mouse' is outside of the mental context of the original paper

Would you be OK with removing it?

Reply:
Or at the very least rephrasing to give more context (e.g., 'equivalent to approaches using dual head-mounted LEDs') or something

Reply:
Also this algorithm is applied to fly data, so is species generic, so specifically mentioning mouse subjects in the description is another thing about it that is a bit weird

Reply:
I think it is fine to remove that addition. I also have the feeling that it confuses more than it clarifies.

Reply:
Btw, it does not mean mice as species but as the stuff that you use to move your pointer in the computer. It is the optical sensor of the (computer) mice that the line is referring to.

Reply:
> Btw, it does not mean mice as species but as the stuff that you use to move your pointer in the computer.

Lol, ahhh.

> I also have the feeling that it confuses more than it clarifies.

Evidently 😆